### PR TITLE
Suppress warnings

### LINKED ISSUE #16: Warnings
Warnings:

    .gem/gems/red-colors-0.3.0/lib/colors/xterm256.rb:45: warning: assigned but unused variable - x
    .gem/gems/red-colors-0.3.0/lib/colors/xterm256.rb:45: warning: assigned but unused variable - y
    .gem/gems/red-colors-0.3.0/lib/colors/xterm256.rb:45: warning: assigned but unused variable - z
    .gem/gems/red-colors-0.3.0/lib/colors/linear_segmented_colormap.rb:83: warning: assigned but unused variable - shape
    .gem/gems/red-colors-0.3.0/lib/colors/colormap_registry.rb:33: warning: assigned but unused variable - existing
    .gem/gems/red-colors-0.3.0/lib/colors/colormap_data/matplotlib_builtin.rb:957: warning: assigned but unused variable - b_data

Not sure if you guys want to fix them. I always use the -w flag when writing ruby code, so it would
be nice if these warnings could be suppressed within the gem. A _ leading underscore should
suffice in these cases - since they are already not used that should be ok.

### PR DESCRIPTION
**Pull request by kou**
fix #16

Reported by rubyFeedback. Thanks!!!

### CODE REVIEW

**Comment by kou** [comment]
@mrkn Can we merge this?

File: lib/colors/colormap_data/matplotlib_builtin.rb [hunk 1]
```diff
@@ -956,7 +956,7 @@ module ColormapRegistry
       a_data = data[a_key]
       b_data = data[b_key]
       a_listed = a_data.is_a?(Hash) && a_data.key?(:listed)
-      b_listed = a_data.is_a?(Hash) && a_data.key?(:listed)
+      b_listed = b_data.is_a?(Hash) && b_data.key?(:listed)
```
Comment (mrkn) [inline lib/colors/colormap_data/matplotlib_builtin.rb:959]: oops!

File: test/test-xterm256.rb [hunk 2]
```diff
@@ -44,33 +44,43 @@ def test_ansi_color_code(code)
...
-          code = 6*(6*r + g) + b + 16
-          red   = (r > 0) ? 40*r + 55 : r
-          green = (g > 0) ? 40*g + 55 : g
-          blue  = (b > 0) ? 40*b + 55 : b
-          label = "Color #{code} is rgb(#{red}, #{green}, #{blue})"
-          data_set[label] = [code, Colors::RGB.new(red, green, blue)]
-        end
+  RGB2GREY = [0.21263900587151035754, 0.71516867876775592746, 0.07219231536073371500]
+
+  # TODO: fix test-unit for keeping data when the data given from a block
```
Comment (mrkn) [inline test/test-xterm256.rb:49]: @kou test-unit doesn't keep data when the data is given from a block.  We must fix it.

Comment (kou) [reply test/test-xterm256.rb:49]: I've fixed it in test-unit 3.5.1.

Comment (mrkn) [reply test/test-xterm256.rb:49]: Thanks!!

**Pull request merged**